package.json

```diff
@@ -143,6 +145,7 @@
     "ts-loader": "^9.2.6",
     "ts-node": "^10.1.0",
     "tsup": "^5.12.8",
+    "tsx": "^3.9.0",
```

Comment:
Can you please share some more details about why `tsx` was needed over `ts-node`?

Reply:
I don't exactly remember. But I'm tired of debugging `ts-node` stuff, so whenever I encounter a problem with it, I just replace the command with `tsx`. Mostly it just works :)
If you want to investigate, you can run `ts-node test/typings/run.ts`.

Reply:
I've tried `ts-node`. There were a few issues that indicated issues with our `test/typings/tsconfig.json`. I've fixed those in 702147f and now the typings tests are passing without issues. I've reverted it to use `ts-node` and deleted `tsx`.

Reply:
Thanks for investigating and fixing it!